integration_test/chrome/capabilities_test.exs

```diff
@@ -0,0 +1,149 @@
+defmodule Wallaby.Integration.CapabilitiesTest do
+  use ExUnit.Case
+  use Wallaby.DSL
+  alias Wallaby.Integration.SessionCase
+  alias Wallaby.Experimental.Selenium.WebdriverClient
+
+  setup do
+    on_exit(fn ->
+      Application.delete_env(:wallaby, :chromedriver)
+    end)
+  end
+
+  describe "capabilities" do
+    test "reads default capabilities" do
+      expected_capabilities = %{
+        javascriptEnabled: false,
+        loadImages: false,
+        version: "",
+        rotatable: false,
+        takesScreenshot: true,
+        cssSelectorsEnabled: true,
+        nativeEvents: false,
+        platform: "ANY",
+        unhandledPromptBehavior: "accept",
```

Comment:
At a glance, some of these options don't seem to be supported in Chromedriver anymore. It would be good to double-check each one of them and remove the unnecessary ones some day (not necessarily in this PR).